auth/src/main/java/com/firebase/ui/auth/ui/accountlink/WelcomeBackPasswordPrompt.java

```diff
@@ -55,7 +56,7 @@
  * the password before initiating a link.
  */
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-public class WelcomeBackPasswordPrompt extends AppCompatBase implements View.OnClickListener {
+public class WelcomeBackPasswordPrompt extends AppCompatBase implements View.OnClickListener, Runnable {
```

Comment:
My comments on this class extend to all the other classes that are using the new listener.